vllm/lora/layers.py

```diff
@@ -54,82 +55,122 @@ def _not_fully_sharded_can_replace(can_replace):
     """
 
     def dec(*args, **kwargs):
-        decorate = kwargs.pop('decorate') if 'decorate' in kwargs else True
-        condition = (not kwargs['lora_config'].fully_sharded_loras
+        decorate = kwargs.pop("decorate") if "decorate" in kwargs else True
+        condition = (not kwargs["lora_config"].fully_sharded_loras
                      if decorate else True)
         return can_replace(*args, **kwargs) and condition
 
     return dec
 
 
-def _apply_lora(
+def _apply_expand(
```

Comment:
let's keep docstrings. also `_apply_expand` is only used in one layer, does it make sense to keep it here since other layers just call `add_expand` directly?